cmd/cosign/cli/env.go

```diff
@@ -21,29 +21,41 @@ import (
 	"strings"
 
 	"github.com/spf13/cobra"
+
+	"github.com/sigstore/cosign/cmd/cosign/cli/options"
+	"github.com/sigstore/cosign/pkg/cosign/env"
 )
 
 func Env() *cobra.Command {
-	return &cobra.Command{
+	o := &options.EnvOptions{}
+
+	cmd := &cobra.Command{
 		Use:   "env",
 		Short: "Prints Cosign environment variables",
 		Args:  cobra.NoArgs,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			for _, e := range getEnv() {
+			// Print COSIGN_ environment variables
+			env.PrintEnv(o.ShowDescriptions, o.ShowSensitiveValues)
+
+			// Print external environment variables (SIGSTORE_ and TUF_)
+			for _, e := range getExternalEnv() {
 				fmt.Println(e)
 			}
+
 			return nil
 		},
 	}
+
+	o.AddFlags(cmd)
+	return cmd
 }
 
-func getEnv() []string {
+func getExternalEnv() []string {
```

Comment:
I think it's worth folding this into `PrintEnv`, there are some subtle scenarios to test.

Reply:
This is done, I'm just not sure if I tested everything expected.

Reply:
I should have written down what I meant, ha. You covered everything I can think of, *except* if we want to make external environment variables "sensitive" by default. Then we need a test case with `--show-sensitive` and one without